src/api/argo-templates/base/iufBase.template.argo.yaml

```diff
@@ -0,0 +1,120 @@
+#
+# MIT License
+#
+# (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: iuf-base-template   # Contains craycli, loftsman, kubectl, git
+  #labels:
+  #  version: "2"
+spec:
+  entrypoint: shell-script
+  templates:
+    - name: shell-script
+      inputs:
+       parameters:
+         - name: scriptContent
+         - name: dryRun
+      retryStrategy:
+       limit: "2"
+       retryPolicy: "Always"
+       backoff:
+         duration: "10s"       # Must be a string. Default unit is seconds. Could also be a Duration, e.g.: "2m", "6h", "1d"
+         factor: "2"
+         maxDuration: "1m"
+      script:
+        # use portainer/kubectl-shell that has:
+        #  - kubectl, craycli, loftsman, git, jq, culr 
```

Comment:
`s/culr/curl/`

Up do you if you want to add `manifestgen` here or not... not sure how long the list might get.

Reply:
I added manifestgen to the list